api/tests/organizational-entities/unit/domain/models/OrganizationLearnerImportFormat.test.js

```diff
@@ -0,0 +1,79 @@
+import { OrganizationLearnerImportFormat } from '../../../../../src/organizational-entities/domain/models/OrganizationLearnerImportFormat.js';
+import { EntityValidationError } from '../../../../../src/shared/domain/errors.js';
+import { expect } from '../../../../test-helper.js';
+
+describe('Unit | Organizational Entities | Domain | Model | OrganizationLearnerImportFormat', function () {
+  describe('#constructor', function () {
+    it('should initialize valid object', function () {
+      //when
+      const organizationLearnerImportFormat = new OrganizationLearnerImportFormat({
+        name: 'SAY_MY_NAME',
+        config: { basic_config: 'toto' },
+        fileType: 'csv',
+      });
+      // then
+      expect(organizationLearnerImportFormat).to.deep.equal({
+        name: 'SAY_MY_NAME',
+        config: { basic_config: 'toto' },
+        fileType: 'csv',
+      });
+    });
+
+    context('Validation Cases', function () {
+      it('return an EntityValidator when missing fileType', function () {
```

Comment:
```suggestion
      it('returns an EntityValidator when missing fileType', function () {
```
pareil pour les tests suivants 😄